src/app/ui/universal-dapp-ui.js

```diff
@@ -131,172 +115,160 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
     contractActionsWrapper.appendChild(this.getCallButton({
       funABI: funABI,
       address: address,
-      contractAbi: contractABI,
+      contractABI: contractABI,
       contractName: contractName
     }))
   })
 
+  const calldataInput = yo`
+    <input id="deployAndRunLLTxCalldata" class="w-100 m-0" title="The Calldata to send to fallback function of the contract.">
+  `
+  const llIError = yo`
+    <label id="deployAndRunLLTxError" class="text-danger"></label>
+  `
+  // constract LLInteractions elements
+  const lowLevelInteracions = yo`
+    <div class="d-flex flex-column">
+      <div class="d-flex flex-row justify-content-between mt-2">
+        <label class="pt-2 border-top d-flex justify-content-start flex-grow-1">
+          Low level interactions with contract
+        </label>
+        <i aria-hidden="true"
```

Comment:
can we have a link to doc instead of this info?

Reply:
ah, sorry about this. totally forgot.